src/status_im/contexts/wallet/events_test.cljs

```diff
@@ -129,38 +127,35 @@
                        [:dispatch [:wallet/get-wallet-token-for-account address-2]]]]
       (is (match? expected-fx result-fx)))))
 
-(deftest get-wallet-token-for-account-test
-  (testing "get wallet token for account"
-    (let [cofx             {:db {}}
-          effects          (events/get-wallet-token-for-account cofx [address])
-          expected-effects {:db {:wallet {:ui {:tokens-loading {address true}}}}
-                            :fx [[:json-rpc/call
-                                  [{:method     "wallet_getWalletToken"
-                                    :params     [[address]]
-                                    :on-success [:wallet/store-wallet-token address]
-                                    :on-error   [:wallet/get-wallet-token-for-account-failed
-                                                 address]}]]]}]
-      (is (match? expected-effects effects)))))
+(h/deftest-event :wallet/get-wallet-token-for-account
+  [dispatcher]
+  (let [expected-effects {:db {:wallet {:ui {:tokens-loading {address true}}}}
+                          :fx [[:json-rpc/call
+                                [{:method     "wallet_getWalletToken"
+                                  :params     [[address]]
+                                  :on-success [:wallet/store-wallet-token address]
+                                  :on-error   [:wallet/get-wallet-token-for-account-failed
+                                               address]}]]]}]
+    (is (match? expected-effects (dispatcher address)))))
 
-(deftest check-recent-history-for-all-accounts-test
+(h/deftest-event :wallet/check-recent-history-for-all-accounts
+  [dispatcher]
   (testing "check recent history for all accounts"
     (let [address-1   "0x1"
           address-2   "0x2"
-          cofx        {:db {:wallet {:accounts {address-1 {:address address-1}
-                                                address-2 {:address address-2}}}}}
-          effects     (events/check-recent-history-for-all-accounts cofx)
-          result-fx   (:fx effects)
           expected-fx [[:dispatch [:wallet/check-recent-history-for-account address-1]]
                        [:dispatch [:wallet/check-recent-history-for-account address-2]]]]
-      (is (match? expected-fx result-fx)))))
+      (reset! rf-db/app-db
+        {:wallet {:accounts {address-1 {:address address-1}
+                             address-2 {:address address-2}}}})
+      (is (match? expected-fx (:fx (dispatcher)))))))
 
-(deftest process-account-from-signal-test
```

Comment:
FYI: I changed a few tests in this namespace, but not all. If the PR is deemed valuable by reviewers, I'll update the entire test file to keep things consistent (should be real quick).